src/common/options/global.yaml.in

```diff
@@ -6439,7 +6504,7 @@ options:
 - name: journal_max_write_entries
   type: int
   level: advanced
-  desc: Max IOs in flight to journal
+  desc: Max IOs in flight to journal (deprecated)
```

Comment:
How did you find this was deprecated? Question goes for all others labeled "deprecated".

Reply:
I applied that to anything related to Filestore.  As I've found references to Filestore (and cache tiers for that matter) in the docs I've been endeavoring to emphasize that they should be eschewed.

Reply:
The config checker surfaced some YAML issues that I already backed out.
Re Teuthology, this is a perfect example of someone (me) who lacks a complete understanding of how / when that happens.  I'm of course onboard with stringent checking but am less clear on when to consider it mergeable, or if someone else will do that once it passes.